src/state/middleware/scigateway.middleware.tsx

```diff
@@ -49,6 +47,10 @@ export const listenToPlugins = (
   dispatch: Dispatch,
   getState: () => StateType
 ): void => {
+  const darkModeLocalStorage = localStorage.getItem('darkMode');
+  const darkModePreference = darkModeLocalStorage === 'true' ? true : false;
```

Comment:
Same as before, might want to use `prefers-color-scheme` - perhaps this tiny bit of common code could be refactored out? Or, if the `prefers-color-scheme` change is tiny perhaps it's not worth it